js/ui/lookup.d.ts

```diff
@@ -378,6 +386,11 @@ export interface dxLookupOptions extends dxDropDownListOptions<dxLookup> {
      * @public
      */
     usePopover?: boolean;
+    /**
+     * @docid dxLookupOptions.dropDownOptions
+     * @type dxPopupOptions | dxPopoverOptions
+     */
+    dropDownOptions?: {};
```

Comment:
Why not `dropDownOptions?: dxPopupOptions | dxPopoverOptions`? @IlyaKhD our tooling accept such cases?

Reply:
You're right. Fixed